torch/ao/sparsity/_variables.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+def get_sparse_mapping():
```

Comment:
Can these be plain dicts instead of functions, to align with quantization (see https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantization_mappings.py#L72)

Reply:
The quantization_mappings also use functions. In fact, using regular dicts is a bad idea, as they are mutable